ksql-engine/src/test/java/io/confluent/ksql/function/UdfArgCoercerTest.java

```diff
@@ -64,14 +78,14 @@ public void testCoerceBoxedNull() {
     Object[] args = new Object[]{null};
 
     // Then:
-    assertThat(UdfTemplate.coerce(args, Integer.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, Long.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, Double.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, String.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, Boolean.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, Map.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, List.class, 0), nullValue());
-    assertThat(UdfTemplate.coerce(args, Object[].class, 0), nullValue());
+    assertThat(UdfArgCoercer.coerceUdfArgs(args, Integer.class, 0), Matchers.nullValue());
```

Comment:
nit: same here - static import for `nullValue` please.

Reply:
done